test/UID2.Client.Test/TestData.cs

```diff
@@ -52,6 +53,42 @@ internal static string KeySetToJson(params Key[] keys)
                 }}")) + "]}";
         }
 
+        internal static string KeySharingResponse()
+        {
+            var keyToJson = (Key key) => new
+            {
+                id = key.Id,
+                site_id = key.SiteId,
+                created = DateTimeUtils.DateTimeToEpochSeconds(key.Created),
+                activates = DateTimeUtils.DateTimeToEpochSeconds(key.Activates),
+                expires = DateTimeUtils.DateTimeToEpochSeconds(key.Expires),
+                secret = Convert.ToBase64String(key.Secret),
+            };
+
+            var response = new
+            {
+                body = new
+                {
+                    // TODO: add headers
+                    keys = new[]
+                    {
+                        keyToJson(MASTER_KEY),
+                        keyToJson(SITE_KEY),
+                    },
+                    site_data = new[]
+                    {
+                        new
+                        {
+                            id = SITE_ID,
+                            domain_names = new[] { "example.com" }
```

Comment:
could we have a second test that has a list of multiple domain names?

Reply:
Added multiple domain names, let me know if it looks OK to you